src/Channel.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Created by kenshoo-linux on 14/07/16.
+ */
+/**
+ * Created by ken-linux2 on 12/07/16.
+ */
+public class Channel {
+
+    private String name;
+    private String channel_campaign_id;
+    private String channel_adgroup_id;
+    private String channel_account_id;
+    private String channel_ad_id;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        if(name==null)
+            throw new RuntimeException("name cannot be null");
+        this.name = name;
+    }
+
+    public String getChannel_campaign_id() {
+        return channel_campaign_id;
+    }
+
+    public void setChannel_campaign_id(String channel_campaign_id) {
+        if(channel_campaign_id==null)
+            throw new RuntimeException("channel campaign id cannot be null");
+        this.channel_campaign_id = channel_campaign_id;
+    }
+
+    public String getChannel_adgroup_id() {
+        return channel_adgroup_id;
+    }
+
+    public void setChannel_adgroup_id(String channel_adgroup_id) {
+        this.channel_adgroup_id = channel_adgroup_id;
+    }
+
+    public String getChannel_account_id() {
+        return channel_account_id;
+    }
+
+    public void setChannel_account_id(String channel_account_id) {
+        if(channel_account_id==null)
+            throw new RuntimeException("channel account id cannot be null");
+        this.channel_account_id = channel_account_id;
+    }
+
+    public String getChannel_ad_id() {
+        return channel_ad_id;
+    }
+
+    public void setChannel_ad_id(String channel_ad_id) {
+        this.channel_ad_id = channel_ad_id;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
As an exercise try to replace with Apache commons EqualsBuilder. The idea know how to generate this code, but you'll have to add a dependency to build.gradle. I'll email you what to add.